diagnostic.md

```diff
@@ -94,4 +97,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+teamMember['hobby'] = 'exercise', teamMember['name'] ='JasonWeeks',teamMember['name']['surname'] = 'Weeks'
```

Comment:
Correct answer:
```
teamMember['name'] = {
   given: 'Jason',
   surname: 'Weeks'
 }
```

To get the answer `JasonWeeks`:
`teamMember.name.given + teamMember.name.surname = 'JasonWeeks`